tempesta_fw/sock_clnt.c

```diff
@@ -92,7 +100,7 @@ tfw_cli_conn_release(TfwConnection *conn)
 {
 	TfwCliConnection *cli_conn = (TfwCliConnection *)conn;
 
-	del_timer_sync(&cli_conn->ka_timer);
+	del_singleshot_timer_sync(&cli_conn->ka_timer);
```

Comment:
Is this a correct use of a timer delete function? The timer is rearmed on `ss_close()` error in `tfw_sock_cli_keepalive_timer_cb()`, and for that reason it's not a single shot timer.